build.gradle

```diff
@@ -182,6 +182,11 @@ subprojects {
 
   task docsJar(dependsOn: javadocJar)
 
+  javadoc {
+    options.charSet = 'UTF-8'
+    options.encoding = 'UTF-8'
```

Comment:
I assume `encoding` being `UTF-8` is what most people expect. I'm not sure where the garbled characters are coming from either, neither of the following turn up any non-ASCII characters:

```
find . -name *.java | xargs grep --color='auto' -P -n "[\x80-\xFF]"
find . -name *.scala | xargs grep --color='auto' -P -n "[\x80-\xFF]"
```